websub-ballerina/sub_listener.bal

```diff
@@ -88,10 +88,10 @@ public class Listener {
     # + callbackUrl - retrieved callback URL
     private function logGeneratedCallbackUrl(string[]|string? name, string callbackUrl) {
         if (name is ()) {
-            log:print("Could not find service-path. Hence generated callback URL [" + callbackUrl + "]");
+            log:print("Autogenerated callback URL " + callbackUrl);
```

Comment:
Is this function handling an error scenario? If so wouldn't it be better to use `log:printError()` instead of `log:print()`. Maybe a better way to log would be:

`log:printError("service-path not defined", callbackUrl = callbackUrl);`

Reply:
* Here it is a warning not an error. Since ballerina does not have `warn` log level we are using `info`.